docs/repo-docs/reference/configuration.mdx

```diff
@@ -633,3 +633,14 @@ Set endpoint for API calls to the remote cache.
 Default: `"https://vercel.com"`
 
 Set endpoint for requesting tokens during `turbo login`.
+
+### `teamId`
+
+The team id of the Remote Cache team.
+Value will be passed as `teamId` in the querystring for all Remote Cache HTTP calls.
+Must start with `team_` or it will not be used.
```

Comment:
Future task: Should the code warn on this?

Reply:
Or we remove this behavior? I do not know the reasoning behind it 🤷

Reply:
Possibly. I think I'd err on caution for now. I don't like changes for auth things without 100% confidence, and we're already making changes. Unless you're 100% on it, I'd just kick the can down the road.